amf-api-contract/shared/src/main/scala/amf/apicontract/internal/spec/async/parser/bindings/channel/IBMMQChannelBindingParser.scala

```diff
@@ -0,0 +1,74 @@
+package amf.apicontract.internal.spec.async.parser.bindings.channel
+
+import amf.apicontract.client.scala.model.domain.bindings.ibmmq.{
+  IBMMQChannelBinding,
+  IBMMQChannelQueue,
+  IBMMQChannelTopic
+}
+import amf.apicontract.internal.metamodel.domain.bindings.{
+  IBMMQChannelBindingModel,
+  IBMMQChannelQueueModel,
+  IBMMQChannelTopicModel,
+  WebSocketsChannelBindingModel
+}
+import amf.apicontract.internal.spec.async.parser.bindings.BindingParser
+import amf.apicontract.internal.spec.async.parser.context.AsyncWebApiContext
+import amf.core.internal.parser.YMapOps
+import amf.core.internal.parser.domain.Annotations
+import org.yaml.model.{YMap, YMapEntry}
+
+object IBMMQChannelBindingParser extends BindingParser[IBMMQChannelBinding] {
+  override def parse(entry: YMapEntry, parent: String)(implicit ctx: AsyncWebApiContext): IBMMQChannelBinding = {
+    val binding = IBMMQChannelBinding(Annotations(entry))
```

Comment:
Same as the rest of the parsers, most of these facet have default values.

Reply:
done, now we're parsing the value or setting the default one where applicable